Makefile

```diff
@@ -81,15 +81,17 @@ image-apiserver:
 		-t ${REGISTRY}/apiserver-$(GOARCH):$(VERSION) \
 		--platform=linux/$(GOARCH) \
 		--build-arg BASEIMAGE=$(BASEIMAGE) \
-		--build-arg BINNAME=apiserver .
+		--build-arg BINNAME=apiserver . \
+		--load
```

Comment:
I think it's better to put the `--load` flag before `.`
eg.
```bash
		-t ${REGISTRY}/apiserver-$(GOARCH):$(VERSION) \
		--platform=linux/$(GOARCH) \
		--load \
		--build-arg BASEIMAGE=$(BASEIMAGE) \
		--build-arg BINNAME=apiserver .
```

Reply:
check again

Reply:
check again @Iceber